### DISCUSSION TITLE
How can I make input model not deferred?

### DISCUSSION THREAD

**Discussion by o-tari** [filamentphp/filament#6634] [Help] [answered]
I don't want input to have defer on the model, how can I disable it without using custom view?
![Screenshot 2023-05-30 at 12 16 55](https://github.com/filamentphp/filament/assets/51799275/6ef894a4-21fe-4f8c-bd8f-11b490a7f7a6)

**Comment by adevade** [comment, marked as answer]
I believe that adding on `->reactive()` to the `Field` makes it not deferred. 

[Filament Docs](https://filamentphp.com/docs/2.x/forms/advanced#dependant-fields--components)

**Comment by o-tari** [comment]
That worked thanks!
